services/slam/slam.go

```diff
@@ -57,10 +57,10 @@ func FromRobot(r robot.Robot, name string) (Service, error) {
 // Service describes the functions that are available to the service.
```

Comment:
[q] Do the collectors in the init need to be updated?

Reply:
![image](https://github.com/viamrobotics/rdk/assets/34897732/91cad5ad-d9b3-4b8c-a224-723702513a4d)

Reply:
yes, done